README.rst

```diff
@@ -259,35 +269,27 @@ Splitting this section into 3 compartments would results in the following paths:
     [2.        , 2.        ]]
 
 
-Dendrogram with synapses
-------------------------
+Plot morphologies with synapses
+-------------------------------
 
-This functionality is available only when the package is installed with **dendrogram** extras:
+This functionality is available only when the package is installed with **plot** extras:
 
 .. code:: bash
 
-    pip install morph-tool
-
-Draw NeuroM dendrogram with synapses on it. Synapses must be represented as a DataFrame. Required
-columns in this dataframe are:
-
-.. code:: python
-
-    from morph_tool import dendrogram
-    required_columns = [dendrogram.SOURCE_NODE_ID, dendrogram.TARGET_NODE_ID,
-                        dendrogram.POST_SECTION_ID, dendrogram.POST_SECTION_POS,
-                        dendrogram.PRE_SECTION_ID, dendrogram.PRE_SECTION_POS]
+    pip install morph-tool[plot]
 
-or equivalently
+Dendrogram
+~~~~~~~~~~
 
-.. code:: python
+Draw NeuroM dendrogram with synapses on it. Synapses must be represented as a DataFrame. See
+`dendrogram <https://morph-tool.readthedocs.io/en/latest/morph_tool.plot.dendrogram.html>`__.
```

Comment:
I couldn't use here the sphinx directive `:py:func:` because it gets rendered by PyPi so I had to put a direct link. For details see https://packaging.python.org/guides/making-a-pypi-friendly-readme/#validating-restructuredtext-markup